packages/react/src/components/Icons/Icons.stories.scss

```diff
@@ -6,7 +6,7 @@
 //
 .demo-icon-example {
   display: flex;
-  width: 10rem;
+  width: 15rem;
```

Comment:
Would it also be helpful to set a `min-width` on the `svg`?

Reply:
@tw15egan that's a good call. better safe than sorry. 👍🏽

Reply:
@tw15egan pushed an update. lmk if you think I should revert the original width to 10rem.

Reply:
Nah, can't hurt to have both. Thanks for fixing this!